scipy/interpolate/_fitpack2.py

```diff
@@ -1576,6 +1582,8 @@ def __call__(self, theta, phi, dtheta=0, dphi=0, grid=True):
             If `grid` is True: evaluate spline at the grid points
             defined by the coordinate arrays theta, phi. The arrays
             must be sorted to increasing order.
+            Note that the axis ordering is inverted relative to
+            the output of meshgrid.
```

Comment:
So it's consistent with `np.meshgrid(...., indexing="ij")`? If this is indeed the case (please do double-check!), it would be better to state it explicitly. Maybe even give an example (which these things lack at the moment).

Reply:
@ev-br Yes, `indexing="ij"` gives the expected behaviour. I've tweaked descriptions of the parameters to mention "default output" and added examples to illustrate.